magenta/models/latent_transfer/train_joint.py

```diff
@@ -27,13 +27,12 @@
 import importlib
 from os.path import join
```

Comment:
Why is from `os.path import join` used here but not in the previous files?  I prefer the full `os.path.join` either way.

Reply:
Weird. I'm not sure how the linter missed this...